feature/google/src/test/java/com/revenuecat/purchases/google/StoreProductTest.kt

```diff
@@ -26,4 +27,34 @@ class StoreProductTest {
         val storeProducts = productDetails.toStoreProducts()
         assertThat(storeProducts.size).isEqualTo(1)
     }
+
+    @Test
+    fun `list of INAPP and SUBS ProductDetails maps to StoreProducts`() {
+        val productDetail1 = mockProductDetails(
+            productId = "iap_1",
+            type = BillingClient.ProductType.INAPP,
+            subscriptionOfferDetails = null)
+        val productDetail2 = mockProductDetails(
+            productId = "sub_1",
+            type = BillingClient.ProductType.SUBS)
+        val productDetails = listOf(productDetail1, productDetail2)
+
+        val storeProducts = productDetails.toStoreProducts()
+        assertThat(storeProducts.size).isEqualTo(2)
+    }
+
+    @Test
+    fun `list of SUBS ProductDetails with multiple subscription offers maps to multiple StoreProducts`() {
```

Comment:
<3 thank you!

a couple other pieces we could test (though doesn't have to be part of this PR, i think with some design decisions unmade earlier we missed them):
* sub period and default option are null for INAPP
* purchaseOptions is empty for INAPP
* onetimeproductprice is null for SUB